stl/inc/format

```diff
@@ -39,9 +39,10 @@
 #define _FORMAT_
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
-#ifndef __cpp_lib_concepts
-#pragma message("The contents of <format> are available only with C++20 concepts support.")
-#else // ^^^ !defined(__cpp_lib_concepts) / defined(__cpp_lib_concepts) vvv
+#ifndef __cpp_lib_format
+#pragma message("The contents of <format> are available only in c++latest mode with concepts support;")
+#pragma message("see https://github.com/microsoft/STL/issues/1814 for details.")

```

Comment:
```suggestion
#pragma message("see https://github.com/microsoft/STL/issues/1814#issuecomment-845572895 for details.")
```

Reply:
Thanks for the idea @statementreply! Because Casey has already ported this to the MSVC-internal repo, and because we already have release notes linking to GH-1814, I've updated that issue's description, adding at the bottom:

> **Update:** We arrived at a plan, see [the comment below](https://github.com/microsoft/STL/issues/1814#issuecomment-845572895).

Hopefully this should make links to GH-1814 comprehensible, and avoid the need to deep-link to the comment.

Reply:
> adding at the bottom

Maybe put it at the top or make it 

# big

So it's more noticeable? I think it could easily be missed as is

Reply:
I threw in a horizontal rule and made the "update" line header3 styled - I think it's now sufficiently visible.

Reply:
I'm considering these comments resolved by STL's changes to #1814. If folks still feel like the "plan" parts of that issue need more visibility, please comment there and we'll address it.